acceptance/suites/tests/authorization/default_rules.rb

```diff
@@ -57,6 +57,8 @@ def report_query(node)
 end
 
 with_puppet_running_on(master, {}) do
+  masterfqdn = on(master, '/opt/puppetlabs/bin/facter fqdn').stdout.chomp
```

Comment:
Should we also use this in the URL hostname which is added to the curl command in `curl_authenticated` and `curl_unauthenticated` instead of `master`?